devops/helm-charts/go-server/charts/postgres-cluster/init.sql

```diff
@@ -0,0 +1,155 @@
+CREATE TABLE public.admins (
+    id bigint NOT NULL,
+    username character varying(64) NOT NULL,
+    password text NOT NULL
+);
+
+ALTER TABLE public.admins ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY (
+    SEQUENCE NAME public.admins_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE public.customers (
+    id bigint NOT NULL,
+    language_id bigint NOT NULL,
+    phone_humber character varying(20) NOT NULL
+);
+
+ALTER TABLE public.customers ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY (
+    SEQUENCE NAME public.customers_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE public.languages (
+    id bigint NOT NULL,
+    short_name character(2) NOT NULL,
+    full_name character varying(32) NOT NULL
+);
+
+ALTER TABLE public.languages ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY (
+    SEQUENCE NAME public.languages_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE public.order_statuses (
+    id bigint NOT NULL,
+    ready_at timestamp without time zone,
+    returned_at timestamp without time zone,
+    customer_notified_at timestamp without time zone,
+    is_outsourced boolean DEFAULT false NOT NULL,
+    is_receipt_lost boolean DEFAULT false NOT NULL
+);
+
+ALTER TABLE public.order_statuses ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY (
+    SEQUENCE NAME public.order_statuses_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE public.order_types (
+    id bigint NOT NULL,
+    short_name character varying(16) NOT NULL,
+    full_name character varying(32) NOT NULL
+);
+
+ALTER TABLE public.order_types ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY (
+    SEQUENCE NAME public.order_types_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1
+);
+
+CREATE TABLE public.orders (
+    id bigint NOT NULL,
+    order_status_id bigint NOT NULL,
+    order_type_id bigint NOT NULL,
+    worker_id bigint NOT NULL,
+    customer_id bigint NOT NULL,
+    reason text,
+    defect text,
+    total_price character varying(64),
```

Comment:
What is the most appropriate data type for total_price? Should we keep it as varchar, or would it be better to change it to a numeric type like float or decimal?

Reply:
i decided to use decimal type instead of varchar